src/main/java/com/redhat/red/build/koji/model/json/util/BuildExtraInfoDeserializer.java

```diff
@@ -53,11 +53,11 @@ public BuildExtraInfo deserialize( JsonParser jp, DeserializationContext ctxt )
                 {
                     case ( MAVEN_INFO ):
                     {
-                        JsonDeserializer<Object> mvnDeser =
-                                ctxt.findRootValueDeserializer( ctxt.constructType( MavenExtraInfo.class ) );
+                        JsonDeserializer<Object> typeInfoDeser =
```

Comment:
You know, 'extra' is a free-form area according to Koji, but I don't think it needs to be for us. We can probably remove these serializer/deserializer classes  (BuildExtraInfoDeserializer/Serializer and TypeInfoDeserializer/Serializer) and just allow Jackson to do it. I think as long as we can handle null values in the event this stuff isn't in the extra map, we should be ok. You can also configure Jackson to ignore fields it doesn't recognize, so we could use that to deal with other data in this section that we don't know how to represent.

I'd be in favor of removing these.